src/quests/src/quests_state.c

```diff
@@ -314,5 +321,13 @@ bool npc_can_give_task(quest_ctx_t *qctx, char *task_id)
     prereq_t *prereq = task->prereq;
     player_t *player = qctx->player;
 
+    player_task_t *ptask = get_player_task_from_hash(task_id, player->player_tasks);
+    if(!ptask) {
```

Comment:
i know that this works but for the sake of clarity:
```suggestion
    if(ptask == NULL) {
```

Reply:
Fixed I think